optimum/utils/normalized_config.py

```diff
@@ -140,6 +140,9 @@ def __getattr__(self, attr_name):
 MPTNormalizedTextConfig = NormalizedTextConfig.with_args(
     num_attention_heads="n_heads", hidden_size="d_model", num_layers="n_layers"
 )
+GPTBigCodeNormalizedTextConfig = NormalizedTextConfig.with_args(
+    num_attention_heads="n_head", hidden_size="n_inner", num_layers="n_layer"
```

Comment:
Isn't it `n_embd` instead of `n_inner` ?
```suggestion
    num_attention_heads="n_head", hidden_size="n_embd", num_layers="n_layer"
```